Tournament.cpp

```diff
@@ -0,0 +1,221 @@
+#include "Tournament.h"
+
+	Tournament::Tournament()
+	{
+		std::string EuroPath = "../FIFApoints/EuroCupParticipants.txt";
+		std::string EuroGroupPath = "../FIFApoints/EuroCupGroups.txt";
+		std::string WorldCupPath = "../FIFApoints/WorldCupParticipants.txt";
+		std::string WorldCupGroupPath = "../FIFApoints/WorldCupGroups.txt";
+		switch (ChooseTournament())
+		{
+		case 1:
+			AddParticipants(EuroPath, m_euroParticipants);
+			SetGroup(EuroGroupPath, m_numberEuroGroup);
+			break;
+		case 2:
+			AddParticipants(WorldCupPath, m_worldCupParticipants);
+			SetGroup(WorldCupGroupPath, m_numberWorldCupGroup);
+			break;
+		default:
+			std::cout << "Default case!" << std::endl;
+			break;
+		}
+	}
+	Tournament::~Tournament() {}
+	int Tournament::ChooseTournament()
+	{
+		int choice;
+		do {
+			std::cout << "---Select a tournament---" << std::endl;
+			std::cout << "Press 1 for Euro or 2 for World Cup!" << std::endl;
+			std::cin >> choice;
+		} while (choice < 1 or choice > 2);
+		return choice;
+	}
+
+	void Tournament::AddParticipants(std::string path, int numberParticipants)
+	{
+		std::ifstream ParticipantsList;
+		ParticipantsList.open(path);
+		if (!ParticipantsList.is_open() and !ParticipantsList.good())
+			std::cout << "File is not open!" << std::endl;
+		else
+		{
+			std::string name;
+			int rank;
+			double fifa_points;
```

Comment:
Better default-initialise them to zero,  otherwise if something goes wrong in the loop below it will be harder to debug.